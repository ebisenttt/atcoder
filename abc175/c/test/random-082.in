935013149346067 8235624260248 397153008170476
