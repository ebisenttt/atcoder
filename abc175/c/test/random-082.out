140707133005115
